spec/presenters/two_factor_authentication/selection_presenter_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe TwoFactorAuthentication::SelectionPresenter do
-  class PlaceholderPresenter < TwoFactorAuthentication::SelectionPresenter
+  class SelectionPlaceholderPresenter < TwoFactorAuthentication::SelectionPresenter
```

Comment:
I would opt to define an anonymous class via `let` instead of fix this one's name?

```suggestion
  let(:placeholder_presenter_class) do
    Class.new(TwoFactorAuthentication::SelectionPresenter) do
```
etc

Reply:
thanks, done in 8012a9c11e